gradle.properties

```diff
@@ -23,6 +23,9 @@ org.gradle.parallel = true
 
 kotlin.code.style = official
 
+# The version of the JDK to use for building ORT.
+javaLanguageVersion = 11
```

Comment:
BTW, I'd be fine with giving this another / shorter name. Maybe just `buildJdk`?

Reply:
I'm fine with either name, but add `javaToolchainVersion` to the candidates.